cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -247,4 +255,12 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
             outputDir.resolve(relativePath.toString()).safeDeleteRecursively()
         }
     }
+
+    private fun archiveAll() {
+        val archiveFile = outputDir.resolve("archive.zip")
+
+        outputDir.packZip(archiveFile) { path ->
+            path != archiveFile.toPath()
```

Comment:
As a small optimization, you could move out the path conversion so it's not called on every lambda invocation by adding

    val zipPath = zipFile.toPath()

to line 261.